qa/tasks/mixed_system_tests/ios.py

```diff
@@ -0,0 +1,145 @@
+"""
+Component IOs
+"""
+
+import yaml
+import logging
+import os
+import pwd
+import time
+
+from teuthology.orchestra import run
+
+log = logging.getLogger(__name__)
+
+
+cephqe_scripts = {
+    "DIR": {"v1":
+                {"script": "rgw/v1/tests/s3/",
+                 "config": "rgw/v1/tests/s3/yamls"},
+            "v2":
+                {"script": "rgw/v2/tests/s3_swift/",
+                 "config": "rgw/v2/tests/s3_swift/configs"}
+            },
+    "MASTER_BRANCH": "master",
+    "REPO_NAME": "ceph-qe-scripts",
+    "WIP_BRANCH": None
+}
+
+
+class rgw_ios:
+    """
+    RGW IOS using ceph-qe-scripts
+    """
+
+    def __init__(self, ctx, config):
+        self.ctx = ctx
+        self.config = config
+
+    def __enter__(self):
+        log.info('starting rgw-tests')
+        log.info('config %s' % self.config)
+        if self.config is None:
+            self.config = {}
+        assert isinstance(self.config, dict), \
+            "task set-repo only supports a dictionary for configuration"
+        config_file_name = self.config['test'] + ".yaml"
+        log.info('test_version: %s' % self.config.get('test_version', 'v2'))
```

Comment:
All those general info logs on the test version and such should most likely go to the parent class.